controller/cartController.js

```diff
@@ -0,0 +1,21 @@
+import { cartService } from '../service';
+
+const getCartItems = async (req, res) => {
+  try {
+    const { id: userId } = req.foundUser[0];
+    const cartItems = await cartService.getCartItems(userId);
+
+    if (!cartItems.length) {
+      return res
+        .status(400)
+        .json({ message: 'CART_ITEM_NOT_FOUND', cartItems });
+    }
```

Comment:
```suggestion
      return res
        .status(404)
        .json({ message: 'CART_ITEM_NOT_FOUND', cartItems });
    }
```
1. 이렇게 데이터가 없을 때는, `404` 상태코드를 주기도 합니다 :)
2. cart item이 없는 경우에도 11번 줄에서 `cartItems`를 return하시는 이유가 있나요?

Reply:
카트 아이템이 없을 땐 빈 배열이 출력되고 있습니다.  cartItems: [] 이런 식으로요!
이렇게 빈 배열을 출력해서 보여주는 것이 명확하다고 생각되어 리턴 해주었는데, 적절하지 않다면 삭제하도록 하겠습니다!

Reply:
@hxyxneee 프론트에서 해당 키값을 일단 받고 있을 수도 있겠군요. 
이 부분은 혹시 프론트에서도 장바구니 없을 때 빈 배열 있는게 편하다고 하면 살려두고, 
없어도 처리 가능하다고 하면 삭제하시죠 👍

Reply:
빈 배열을 출력해서 보여주는 게 조금 명확하다는 생각이 들어, 우선 살려두었습니다 소헌님 !